ui/pages/extensions.js

```diff
@@ -94,7 +94,57 @@ const MeshMapSignUpcard = ({ classes, hasAccessToMeshMap=false }) => {
   )
 }
 
+const LogoStyles = () => ({
+
+  img : {
+    paddingRight : "1rem",
+    height : "auto",
+    width : "auto",
+    maxWidth : "220px",
+    maxHeight : "150px",
+  },
+})
+
+const MeshMapSnapShotLogo = withStyles(LogoStyles)((classes) => {
+  return <img className={classes.img} src="/static/img/meshmap-snapshot-logo.svg" />
+})
+
+const MeshMapSnapShotCard = ({ classes,githubActionEnabled=false }) => {
+
+  const handleEnable = (e) => {
+    window.open("https://cloud.meshery.io/connect/github/new/", "_blank")
+    e.stopPropagation();
+  };
+
+  return (
+    <Grid item {...INITIAL_GRID_SIZE}>
```

Comment:
Is `INITIAL_GRID_SIZE` defined somewhere?

Reply:
Yeah line 64 .